nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -366,11 +367,16 @@ def _get_stat_collector(
         """
         range_estimator_params = self._get_range_estimator_parameters(target_point, quantizer_config)
 
+        collector_params = RangeInitCollectorParams(
+            is_weights=target_point.is_weight_target_point(),
+            mode=quantizer_config.mode,
+            per_channel=quantizer_config.per_channel,
+        )
```

Comment:
As I can see, we calculate `RangeInitCollectorParams` instead of using `QuantizerConfig` here. Why do we need that? How does it help?
Based on the ticket, I understood that `RangeInitCollectorParams` calculates in the Torch backend as well and we can re-use it. But it's not, based on the PR's code.

Reply:
The main reason behind is to [calculate parameter `use_abs_max` in one place](https://github.com/openvinotoolkit/nncf/pull/2371#issue-2067303016). And as this `RangeInitCollectorParams` are used in Torch to calculate them, I decided to align all backends and use its logic in OV and ONNX as well.

Alternative solution is to implement new way of `use_abs_max` processing and migrate legacy torch code on it, but I believe this approach will do better